backend/project/urls.py

```diff
@@ -30,5 +30,10 @@
     # url(r'^', include(router.urls)),
     # url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     path('admin/', admin.site.urls),
-    path('api/', include('api.urls'))
+    path('api/', include('api.urls')),
+    url(r'^auth/', include('rest_framework_social_oauth2.urls')),
+]
+
+urlpatterns += [
```

Comment:
Why not add this to the urlpatterns block from line 29?

Reply:
@ckirkwood94 same here

Reply:
@VeraSimon here's the reasoning:[]( https://files.slack.com/files-pri/T4JUEB3ME-FE7DS3N5C/untitled.png)

Reply:
This link just keeps bringing me to the Announcements channel. Can you host it elsewhere and/or just copy any relevant text?